pkgs/development/coq-modules/autosubst/default.nix

```diff
@@ -1,15 +1,18 @@
-{ lib, mkCoqDerivation, coq, mathcomp-ssreflect, version ? null }:
+{ lib, mkCoqDerivation, coq, mathcomp-ssreflect,
+  version ? null, origin ? null }:
 with lib;
 
 mkCoqDerivation {
-  pname = "autosubst";
+  pname   = "autosubst";
+  owner   = "uds-psl";
```

Comment:
This repo redirects to the coq-community one, so this addition is not useful.

Reply:
```suggestion
```